app/database/models.py

```diff
@@ -42,6 +42,7 @@ class User(Base):
     avatar = Column(String, default="profile.png")
     telegram_id = Column(String, unique=True)
     is_active = Column(Boolean, default=False)
+    privacy = Column(String, default="Private", nullable=False)
```

Comment:
If the only values you expect are "private" and "public", consider using boolean type - True ( for private) and Falsr (for public).

Reply:
Originally the feature was planned to contain "friends" as a level of privacy.
This feature has yet to be approved or implemented but I think a room for this option and other levels of privacy can be allowed

Reply:
So maybe use Enum?

Reply:
As this is not a free text field but rather something from a set of closed options in a dropdown I don't think there is any reason to fear an incorrect value here
 Also I am not sure whether in the setting here the solution would be Enum rather than another connecting table like they did in language. In any case I dont think that it is relevant in the situation here right now

Reply:
I think that it's a good practice to add a validation also in backend and in db.
Also, I think that you can get better performance with this option (but I guess we are not worried about that..).

I agree that right now it's too much to add another table.

Please make sure that it's invalid to add a bad value, even if someone tries :)

Reply:
where would you suggest doing this? I didn't see that anyone did something similar  I think... It can't be in the models file obviously.. and as I mentioned no one can enter values I did not specify in the drropdown field, it's not a free text where anyone can enter what they want, just a set of options. Does that not qualify as verifying that only possible values can be entered?

Reply:
so anything? or can I resolve this? :)